tsk/auto/db_postgresql.cpp

```diff
@@ -1048,40 +1043,74 @@ int TskDbPostgreSQL::addFsFile(TSK_FS_FILE * fs_file,
 }
 
 
-int TskDbPostgreSQL::addMACTimeEvent(char*& zSQL, const int64_t data_source_obj_id, const int64_t obj_id, time_t time,
-                                     const int64_t sub_type, const char* full_description)
+int TskDbPostgreSQL::addMACTimeEvents(char*& zSQL, const int64_t data_source_obj_id, const int64_t file_obj_id,
+                                      std::map<int64_t, time_t> timeMap, const char* full_description)
```

Comment:
refactored to support sharing one description for all filesystem events for the same file.